### PR TITLE
OKTA-495502: update to sdk v3

### PR DESCRIPTION
**Pull request by bryanapellanes-okta**

### CODE REVIEW

File: okta-hosted-login/okta-aspnet-mvc-example/okta-aspnet-mvc-example.csproj [hunk 1]
```diff
@@ -258,16 +284,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
         <WebProjectProperties>
-          <UseIIS>True</UseIIS>
-          <AutoAssignPort>True</AutoAssignPort>
-          <DevelopmentServerPort>63475</DevelopmentServerPort>
-          <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>https://localhost:44314/</IISUrl>
```
Comment (laura-rodriguez) [inline okta-hosted-login/okta-aspnet-mvc-example/okta-aspnet-mvc-example.csproj:265]: Did you mean removing these config lines?

File: resource-server/okta-aspnet-webapi-example/okta-aspnet-webapi-example.csproj [hunk 2]
```diff
@@ -206,16 +219,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
         <WebProjectProperties>
-          <UseIIS>True</UseIIS>
```
Comment (laura-rodriguez) [inline resource-server/okta-aspnet-webapi-example/okta-aspnet-webapi-example.csproj:209]: Did you mean removing these lines?

**Review by laura-rodriguez — APPROVED** [review]
We might need to update the changelogs as well. I left a few comments, but other than that LGTM 👍

File: okta-hosted-login/okta-aspnet-mvc-example/okta-aspnet-mvc-example.csproj (hunk 1 above)
Comment (bryanapellanes-okta) [reply okta-hosted-login/okta-aspnet-mvc-example/okta-aspnet-mvc-example.csproj:265]: Yes, these settings caused conflicts with other samples which lead to errors in IIS Express.  Removing them allows the settings to be unique per user without conflicts and errors.

**Pull request merged**